bower.json

```diff
@@ -15,10 +15,15 @@
     "lodash": "~2.4.1",
     "angular-ui-router": "~0.2.10",
     "angular-bootstrap": "~0.11.0",
-    "codemirror": "~5.1.0"
+    "codemirror": "~5.1.0",
+    "angular-file-upload": "~1.1.5",
+    "ng-file-upload": "~3.3.1",
+    "ng-file-upload-shim": "~3.3.1"
```

Comment:
Are we actually using all three of these? Couldn't tell from the code but just want verification